mkdocs.yml

```diff
@@ -166,7 +166,7 @@ nav:
   - Changelog: changelog.md
 
 extra:
-  latest_version: 1.0.0
+  latest_version: 1.0.0-rc1
```

Comment:
I'm not sure if this influences JanusGraph documentation generation or not. I do remember that it has some specific format like `x.y.z` but not sure about `x.y.z-rc1` format. Probably documentation generation should be fine but I think @farodin91 knows better about the docs generation process, so I will rely on his comments to this.

Reply:
I just [searched where it's being used](https://github.com/JanusGraph/janusgraph/search?q=latest_version) and there are some places in the docs. These fall in 3 categories:
1. Name of the distribution build: `janusgraph-{{ latest_version }}.zip`
2. As the version of the JARs to depend on
3. In the URL for Java docs.

The zip archive should include this version and the JARs will also have it so points 1 and 2 are fine. I would also expect the deployment to publish the Java docs although I'm not 100% sure on this. However, there are only 2 links in the docs to the Java docs so it shouldn't be a big problem.

So I'll go ahead now and merge this to proceed with the deployment.